ui/packages/catalog-ui-search/src/main/webapp/react-component/container/results-export/results-export.tsx

```diff
@@ -171,9 +171,15 @@ class ResultsExport extends React.Component<Props, State> {
       })
     } else {
       const source = this.state.selectedResults[0].source
-      const metacardId = this.state.selectedResults[0].id
-
-      response = await exportResult(source, metacardId, transformerId)
+      const uriEncodedSource = encodeURIComponent(source)
+      const metacardId: any = this.state.selectedResults[0].id
+      const uriEncodedMetacardId: any = encodeURIComponent(metacardId)
+      const uriEncodedTransformerId = encodeURIComponent(transformerId)
```

Comment:
can this same issue occur [here](https://github.com/codice/ddf/blob/cbdc57ff14e2de8b2525f60778904cb5ff8bafa8/ui/packages/catalog-ui-search/src/main/webapp/react-component/container/results-export/results-export.tsx#L158) or [here](https://github.com/codice/ddf/blob/cbdc57ff14e2de8b2525f60778904cb5ff8bafa8/ui/packages/catalog-ui-search/src/main/webapp/react-component/container/results-export/results-export.tsx#L167)? Seems like you may want to encode this earlier in the method [here](https://github.com/codice/ddf/blob/cbdc57ff14e2de8b2525f60778904cb5ff8bafa8/ui/packages/catalog-ui-search/src/main/webapp/react-component/container/results-export/results-export.tsx#L138)

Reply:
Yes, updated.